meta-swupd/recipes-core/swupd-client/os-release.bbappend

```diff
@@ -0,0 +1,13 @@
+# swupd-client checks VERSION_ID, which must match the OS_VERSION
+# used for generating swupd bundles in the current build.
+VERSION_ID = "${OS_VERSION}"
+
+FILES_${PN}_append = " /usr/lib/os-release "
+
+# As swupd-client doesn't update files in /etc move os-release's
+# content to /usr/lib
+do_install_append() {
```

Comment:
I'm okay with keeping the do_install_append based on the explanation.

One could also have argued that this should be handled instead by the "stateless" work which is on-going, but for now, let's keep this workaround.